website/docs: docs about initial perms (#14263)
goauthentik/authentik: website/docs/users-sources/access-control/manage_permissions.md

description: "Learn how to use global and object permissions in authentik."
---

> Refer to the following topics for instructions to view and manage permissions. To learn more about the concepts and fundamentals of authentik permissions, refer to [About Permissions](./permissions.md).

Reviewer: ```suggestion
For instructions on viewing and managing permissions, see the following topics.To learn more about the concepts and fundamentals of authentik permissions, refer to [About Permissions](./permissions.md).
```

Reviewer: Not sure, as you wish

Author: Nah this is a good edit! (Always start with the task, the thing to do...)

6. In the **Assign permission to user** box, click the plus sign (**+**) and then click the checkbox beside each permission that you want to assign to the user. To remove permissions, deselect the checkbox.
7. Click **Add**, and then click **Assign** to save your changes and close the box.

> ### Assign 'Can view Admin interface` permissions'

Reviewer: ```suggestion
### Assign `Can view Admin interface` permissions
```

What is up with markdown here? Like this IG?

Author: ooooffff that's my bad typing, thanks for catching!

> You can grant regular users, who are not superusers nor Admins, the right to view the Admin interface. This can be useful in scenarios where you have a team who needs to be able to create certain objects (flows, other users, etc) but who should not have full access to the Admin interface.

Reviewer 2: > not superusers nor Admins

What do you mean by each term? Is there a difference between these?

Author: Ummm I am actualy not sure but I think so.. a user can be a super-user b ut not an administrator , i.e.not part of the akadmin group...?

To assign the `Can view Admin interface` permission to a user (follow the same steps for a role):

1. Go to the Admin interface and navigate to **Directory -> User**.
2. Select a specific user the clicking on the user's name.
3. Click the **Permissions** tab at the top of the page.
4. Click **Assigned Global Permissions** to the left.
5. In the **Assign permissions** area, click **Assign Permission**.
6. In the **Assign permission to user** box, click the plus sign (**+**), enter `admin` in the Search field and click the search icon.
7. Select the returned permission, click **Add**, and then click **Assign** to save your changes and close the box.

> Be aware that any rights beyond viewing will need to be assigned as well; for example, if you want a non-Admin user to be able to create flows in the Admin interface, you need to grant those object permissions.

Reviewer 2: ```suggestion
Be aware that any rights beyond viewing will need to be assigned as well; for example, if you want a non-Admin user to be able to create flows in the Admin interface, you need to grant those permissions.
```

(This is a technicality, `add_*` permissions are only useful when they're global.)

Author: not sure I get this.. there are object-level perms that are not global

Author: Got it, thanks!
